cumulusci/tasks/robotframework/tests/test_browser_proxies.py

```diff
@@ -0,0 +1,137 @@
+import io
+from unittest import mock
+
+from cumulusci.tasks.robotframework import debugger
+from cumulusci.tasks.robotframework.debugger.ui import (
+    BrowserProxy,
+    SeleniumProxy,
+    initialize_highlight_js,
+)
+
+"""
+These tests verify that the proxy used by the debugger uses
+the appropriate javascript methods for the given browser library
+"""
+
+
+class StringContaining(str):
+    """Used for fuzzy assertions"""
+
+    def __eq__(self, other):
+        return self in other
+
+
+class TestSeleniumProxy:
+    @classmethod
+    def setup_class(cls):
+        cls.selenium = mock.Mock()
+        cls.proxy = SeleniumProxy(cls.selenium)
+
+    def setup_method(self):
+        self.mock_listener = mock.Mock()
+        self.mock_builtin = mock.Mock()
+        self.stdout = io.StringIO()
+        self.cli = debugger.DebuggerCli(self.mock_listener, stdout=self.stdout)
+        self.cli.builtin = self.mock_builtin
+        self.selenium.reset_mock()
+
+    def test_webbrowser_browser(self):
+        """Verify the cli has initialized the webbrowser attribute to BrowserProxy
+
+        ... but only Browser has been loaded and SeleniumLibrary has not
+        """
+        self.cli.builtin.get_library_instance = mock.Mock(
+            return_value={
+                "SeleniumLibrary": mock.Mock(),
+                "Browser": mock.Mock(),
+            }
+        )
+        assert isinstance(self.cli.webbrowser, SeleniumProxy)
```

Comment:
I don't understand this assertion vis-a-vis the docstring. Is the docstring a copy/paste mistake?

Reply:
Ah, good catch! The docstring is incorrect. I'll fix it.